lang/python/django-simple-captcha/Makefile

```diff
@@ -8,13 +8,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=django-simple-captcha
-PKG_VERSION:=0.5.9
-PKG_RELEASE:=2
+PKG_VERSION:=0.5.11
+PKG_RELEASE:=1
+
 PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Eneas U de Queiroz <cote2004-github@yahoo.com>
```

Comment:
This should go above PKG_LICENSE I think.

Reply:
Just out of curiosity, where did you get that from?

Reply:
Most packages in the normal OpenWrt repository are like this.

Reply:
That's a funny reason to request a change :laughing:.  I did it nonetheless; it helped me to catch the use of brackets instead of parentheses just above.